packages/react-dom/src/events/accumulateEventTargetListeners.js

```diff
@@ -32,7 +34,8 @@ export default function accumulateEventTargetListeners(
           const listener = captureListeners[i];
           const {callback} = listener;
           dispatchListeners.push(callback);
-          dispatchInstances.push(targetContainer);
+          dispatchInstances.push(null);
```

Comment:
Why are these null?

Reply:
We don't have an instance for `EventTarget` listeners, because they don't need or might even have a backing fiber (i.e. `window` or `document`).